libraries/ui/src/Footer.tsx

```diff
@@ -11,73 +11,105 @@ export type FooterProps = React.PropsWithChildren<{
   logo?: string
 }>;
 
+type FooterSectionProps = {
+  title?: string;
+  links?: { href: string; label: string }[];
+};
+
+const FooterLinksSection: React.FC<FooterSectionProps> = ({ title, links }) => (
+  <div className="footer__section">
+    {title && <p className="footer__heading font-[650] text-on-dark mb-4">{title}</p>}
+    {links && (
+      <ul className="footer__list space-y-2 mb-auto font-normal list-none p-0">
```

Comment:
Do we need `list-none` here? Looks like that is the default for tailwind `ul`

Reply:
We have this in our globals.css:
```css
ul {
    @apply list-inside list-disc;
  }
```
So it does have indentation without `list-none`:
![Screenshot 2025-02-07 at 20 05 56](https://github.com/user-attachments/assets/3038716f-95b0-4861-89d0-5d277372195f)